packages/module/patternfly-docs/content/extensions/chatbot/examples/demos/ChatbotDisplayMode.tsx

```diff
@@ -0,0 +1,459 @@
+                      </DropdownItem>
+                      <DropdownItem
+                        value={ChatbotDisplayMode.drawer}
+                        key="switchDisplayDock"
+                        icon={<OpenDrawerRightIcon aria-hidden />}
+                        isSelected={displayMode === ChatbotDisplayMode.drawer}
+                      >
+                        <span>Dock to window</span>
+                      </DropdownItem>
+                      <DropdownItem
+                        value={ChatbotDisplayMode.fullscreen}
+                        key="switchDisplayFullscreen"
+                        icon={<ExpandIcon aria-hidden />}
+                        isSelected={displayMode === ChatbotDisplayMode.fullscreen}
+                      >
+                        <span>Fullscreen</span>
+                      </DropdownItem>
+                    </DropdownList>
+                  </DropdownGroup>
+                </ChatbotHeaderOptionsDropdown>
+                {(displayMode === ChatbotDisplayMode.drawer || displayMode === ChatbotDisplayMode.fullscreen) && (
+                  <ChatbotHeaderCloseButton onClick={handleChatbotVisibilityDrawer} />
+                )}
+              </ChatbotHeaderActions>
+            </ChatbotHeader>
+            <ChatbotContent>
+              <MessageBox announcement={announcement}>
+                <ChatbotWelcomePrompt
+                  title="Hello, Chatbot User"
+                  description="How may I help you today?"
+                  prompts={welcomePrompts}
+                />
+                {messages.map((message, index) => {
+                  if (index === messages.length - 1) {
+                    return (
+                      <React.Fragment key={message.id}>
+                        <div ref={scrollToBottomRef}></div>
+                        <Message {...message} />
+                      </React.Fragment>
+                    );
+                  }
+                  return <Message key={message.id} {...message} />;
+                })}
+              </MessageBox>
+            </ChatbotContent>
+            <ChatbotFooter>
+              <MessageBar onSendMessage={handleSend} hasMicrophoneButton isSendButtonDisabled={isSendButtonDisabled} />
+              <ChatbotFootnote {...footnoteProps} />
+            </ChatbotFooter>
+          </>
+        }
+      />
+    </Chatbot>
+  );
+
+  const chatbotToggle =
+    displayMode === ChatbotDisplayMode.default ? (
+      <ChatbotToggle
+        tooltipLabel="Chatbot"
+        isChatbotVisible={chatbotVisible}
+        onToggleChatbot={() => {
+          setChatbotVisible(!chatbotVisible);
+        }}
+        id="chatbot-toggle"
+        ref={toggleRef}
+      />
+    ) : null;
+
+  const panelContent = (
+    <DrawerPanelContent>
+      {chatbotComponent}
+      <DrawerActions>
+        <ChatbotHeaderCloseButton onClick={() => setIsDrawerOpen(false)} />
+      </DrawerActions>
+    </DrawerPanelContent>
```

Comment:
I don't think we need DrawerActions or the CloseButton here - we have it elsewhere in the demo.